target-platform/target-platform.target

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><?pde version="3.8"?>
-<target name="wildwebdeveloper" sequenceNumber="10">
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><?pde version="3.8"?><target name="wildwebdeveloper" sequenceNumber="10">
```

Comment:
Useless change, please try to avoid it to reduce noise and make discussions easier to focus on actual payload.